### PR TITLE
Add formatter for `.cabal` and `.nix` files

### LINKED ISSUE #100: Add cabal-fmt and nixpkgs-fmt to `bin/format`
```
nixpkgs-fmt *.nix
cabal-fmt -i *.cabal
```

### PR DESCRIPTION
**Pull request by sergesku**
This resolves #100

### CODE REVIEW

File: plutarch.cabal [hunk 1]
```diff
@@ -63,88 +59,93 @@ common c
...
-    NoStarIsType
     TraditionalRecordSyntax
     TupleSections
     TypeApplications
+    TypeFamilies
     TypeOperators
     TypeSynonymInstances
+    ViewPatterns
+
+  -- poor man's GHC2021
```
Comment (TotallyNotChase) [inline plutarch.cabal:70]: nit: This comment is out of context now, probably could be removed.

**Review by TotallyNotChase — APPROVED** [review]
Looks good!

File: plutarch.cabal (hunk 1 above)
Comment (sergesku) [reply plutarch.cabal]: :ok_hand:

**Comment by L-as** [comment]
Please rebase onto staging.

**Comment by L-as** [comment]
?

**Pull request closed without merging**